README.md

```diff
@@ -54,7 +54,7 @@ Your result will look like:
 ...
 ```
 
-**think**: Why does 03/13 come out as _sad_ when it should be _happy_? How could we fix this?
+**think**: Why does 03/13 come out as _neutral_ when it should be _happy_? How could we fix this? because it can't comprehend punctuation in the .include method. We fix it by removing all punctuation in the happy/sad array and in the input string
 
 2. To make the results a little more accurate, let's write and utilize a method called `strip_punctuation` to strip out the punctuation that affects the results. Namely, remove  exclamation marks (!), periods (.), commas (,), and hashtags (#).
 
```

Comment:
Where are the rest of your answers to the think questions? I wanted to see your thought process :(